concent-builder/configure-jenkins.yml

```diff
@@ -0,0 +1,184 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+    - ../containers/versions.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: jenkins
+      block:
+        - name: Clone concent repositories
+          git:
+            repo:    "{{ item.url }}"
+            dest:    "{{ jenkins_home_dir }}/{{ item.name }}"
+            clone:   yes
+            update:  yes
+            version: dev
+            force:   yes
+          with_items:
+          #  - { url: "{{ repositories['concent-deployment'].url }}", name: concent-deployment }
```

Comment:
Why is it commented-out?

Reply:
I used forked repository for github webhook tests. Please read description of pull request.

Reply:
It's not that obvious that this is specifically the part the description is talking about. And anyway, there's more than one such place in the code. It would be better if you marked things like this with something like `# TMP:` in the code with a comment that explains it. It would make those places easier to find and would also make the purpose clear in the review.